microsite-next/package.json

```diff
@@ -17,17 +17,24 @@
     "docusaurus": "docusaurus"
   },
   "devDependencies": {
+    "@docusaurus/module-type-aliases": "2.3.0",
     "@spotify/prettier-config": "^14.0.0",
+    "@tsconfig/docusaurus": "^1.0.6",
+    "@types/webpack-env": "^1.18.0",
     "js-yaml": "^4.1.0",
     "prettier": "^2.6.2",
-    "replace": "^1.2.2"
+    "replace": "^1.2.2",
+    "typescript": "^4.9.4",
+    "yaml-loader": "^0.8.0"
   },
   "prettier": "@spotify/prettier-config",
   "dependencies": {
-    "@docusaurus/core": "2.2.0",
-    "@docusaurus/preset-classic": "2.2.0",
+    "@docusaurus/core": "2.3.0",
+    "@docusaurus/preset-classic": "2.3.0",
     "clsx": "^1.1.1",
+    "docusaurus-plugin-sass": "^0.2.3",
```

Comment:
I'm thinking that it might be easier if we can just copy the CSS as it is rather than us supporting `scss`? Just feeling that it's another abstraction ontop of the microsite, and it's more effort to be refactoring into `scss`? I'm not adverse to having support for it, I quite like it, just conscious that it's going to add another layer of complexity to get the work done.

Reply:
Honestly I don't remember why I added it, probably from some example in the docs when trying to use Infima properly (imho it is just a bad clone of Bootstrap for Docusaurus), so made use of it as much as possible, I have already written almost all styles in `SCSS`, `CSS` is valid `SCSS` as well though, just let me know if I should just go back to `CSS`